apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget.js

```diff
@@ -21,35 +22,36 @@
     return (new LockScreenClockWidgetSetup(this));
   };
 
+  LockScreenClockWidget.prototype.updateFormatters = function() {
```

Comment:
Oh sorry forget that. I now see you use it in the ticking state as well. Just keep it here.